group/assignment6/RearrangeCars.cpp

```diff
@@ -127,7 +107,7 @@ void RearrangeCars::generateSequenceOfMovesWithConstraints(const vector<int> &en
 	while (true) {

```

Comment:
numberOfCarsOnDesiredPosition why call this function? how can it be something other than 0 here? I don't get the idea :) please explain

Reply:
If endPosition is the same as startPosition for any car. When number of cars on desired position equals total number of cars, we know that we have to finish an algorithm.